src/main/services/draftService.ts

```diff
@@ -1,11 +1,28 @@
 import * as config from 'config'
 import request from 'client/request'
-import { DraftService as BaseDraftService } from '@hmcts/draft-store-client'
+import { Draft, DraftService as BaseDraftService, Secrets } from '@hmcts/draft-store-client'
 
 import { ServiceAuthTokenFactoryImpl } from 'common/security/serviceTokenFactoryImpl'
 
 export class DraftService extends BaseDraftService {
+  private secrets?: Secrets
+
   constructor () {
     super(config.get<string>('draft-store.url'), request, new ServiceAuthTokenFactoryImpl())
+
+    const primarySecret: string = config.get<string>('draft-store.secrets.primary')
+    const secondarySecret: string = config.get<string>('draft-store.secrets.secondary')
+
+    if (primarySecret) {
+      this.secrets = new Secrets(primarySecret, secondarySecret)
+    }
```

Comment:
any tips on how to test this?

Reply:
any reason to make it optional in the first place?